docs/ref/checks.txt

```diff
@@ -141,6 +141,8 @@ If you're using MySQL, the following checks will be performed:
   ``max_length`` > 255.
 * **mysql.W002**: MySQL Strict Mode is not set for database connection
   '<alias>'. See also :ref:`mysql-sql-mode`.
+* **mysql.W003**: MySQL does not support a database index on
```

Comment:
I don't understand why you choose to have a database-specific exception for MySQL but not for Oracle? You should think about how to make it easy for third-party database backends to pass the test. I guess sticking to `fields.W162` will make that easier. Maybe adding a `DatabaseWrapper.display_name` property will help mentioning the database in the message.